### PR TITLE
feat: add online offline consistency validation script

### PR DESCRIPTION
**Pull request by oh2024**
* **What kind of change does this PR introduce?** 
Add scripts to validate consistency between online and offline

### CODE REVIEW

File: tools/validation/test.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import subprocess
+import os
+import csv
+
+# Execute the offline SQL command
+subprocess.run("/work/openmldb/bin/openmldb --zk_cluster=127.0.0.1:2181 --zk_root_path=/openmldb --role=sql_client < offline.sql",
```
Comment (tobegit3hub) [inline tools/validation/test.py]: It would be better to make this binary configurable.

File: tools/validation/test_consistency.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/work/openmldb/bin/openmldb --zk_cluster=127.0.0.1:2181 --zk_root_path=/openmldb --role=sql_client  < /tmp/prepare_data.sql
```
Comment (tobegit3hub) [inline tools/validation/test_consistency.sh]: Add README.md to introduce the usage.

**Review by tobegit3hub — CHANGES_REQUESTED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by zhanghaohit — APPROVED** [review]
LGTM

**Pull request merged**